boards/samr34-xpro/include/board.h

```diff
@@ -76,6 +77,14 @@ extern "C" {
 #define BTN0_MODE                   GPIO_IN_PU                          /**< Pull Up GPIO   */
 /** @} */
 
+/**
+ * @name MTD configuration
+ * @{
+ */
+extern mtd_dev_t *mtd0;
```

Comment:
```suggestion
extern mtd_dev_t *mtd0; /**< First memory type device */
```

The CI doc-check is insisting on having this documented. We should try to figure out a way to teach the CI when additional documentation is not really helpful ...